config-overrides.js

```diff
@@ -1,8 +1,5 @@
-const { override, addBundleVisualizer } = require('customize-cra')
+const { override } = require('customize-cra')
 const { addReactRefresh } = require('customize-cra-react-refresh')
 
 /* config-overrides.js */
-module.exports = override(
-  addReactRefresh({ disableRefreshCheck: true }),
```

Comment:
why are we disabling refresh?

Reply:
I am not disabling it bro. I am adding react- refresh to the project. Only the refresh-check is disabled. The configuration setup is done with the help of https://github.com/esetnik/customize-cra-react-refresh